src/server/routes/api.ts

```diff
@@ -172,47 +159,35 @@ connection.once('open', () => {
                     break;
             }
         });
-        await allTissues.sort();
-        await allModels.sort();
 
         //////////////////////////////////////////////////////////////////////////////////////////
     });
 
-    GenesInfo.find({ nominations: { $gt: 0 } })
+    GenesInfo.find({ nominations: { $gt: 0 } }).lean()
```

Comment:
Not super familar with `lean` (or mongoose in general) but seems safe to me since these are all getters and there are no setters

Reply:
I haven't didn't see any custom getters in the code, and the branch looks fine when I run locally...I'm not sure about the TypeScript question.